components/lib/password/Password.vue

```diff
@@ -277,6 +277,17 @@ export default {
         onMaskToggle() {
             this.unmasked = !this.unmasked;
         },
+        onToggleMaskKeyDown(event) {
+            switch (event.code) {
+                case 'Enter':
```

Comment:
I think you need `NumpadEnter` here too since you are using `code`

Reply:
Done @melloware